src/languages/es.js

```diff
@@ -2064,4 +2064,9 @@ export default {
     stateSelectorModal: {
         placeholderText: 'Buscar para ver opciones',
     },
+    demos: {
+        saastr: {
+            signInWelcome: '',
```

Comment:
asking for translation: https://expensify.slack.com/archives/C21FRDWCV/p1692091417354539